pygfx/objects/_more.py

```diff
@@ -35,8 +37,8 @@ class Scene(Group):
 
     """
 
-    def __init__(self):
-        super().__init__()
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
this sneaks in the ability to start the scene off as "invisible" too. Not sure if this is a welcome change or not.

Reply:
Looks reasonable to me!